packages/react-components/tests/browser/MessageThread.spec.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { expect } from '@playwright/experimental-ct-react';
+import { test as betaTest } from './FlavoredBaseTest';
+import { Message, MessageThread } from '../../src';
+
+betaTest.describe('MessageThread inline image tests', () => {
+  betaTest.skip(({ isBetaBuild }) => !isBetaBuild, 'The tests should be run for beta flavor only');
+
+  betaTest('MessageThread inline image should show grey box when loading', async ({ mount }) => {
+    const component = await mount(<MessageThread userId={'1'} messages={getMessages('')} />);
+    await component.evaluate(() => document.fonts.ready);
+    await expect(component).toHaveScreenshot('message-thread-inline-image-loading-stage.png');
+  });
+
+  betaTest('MessageThread inline image should show broken image icon with invalid src', async ({ mount }) => {
+    const component = await mount(<MessageThread userId={'1'} messages={getMessages('http://')} />);
+    await component.evaluate(() => document.fonts.ready);
+    await expect(component).toHaveScreenshot('message-thread-inline-image-broken-image.png');
+  });
+
+  const getMessages = (imgSrc: string): Message[] => {
+    const messages: Message[] = [
+      {
+        messageType: 'chat',
+        senderId: 'user3',
+        content: `<p>How should I design my new house?</p><p><img alt="image" src="${imgSrc}" itemscope="png" width="374.53183520599254" height="250" id="SomeImageId2" style="vertical-align:bottom"></p><p>&nbsp;</p>`,
```

Comment:
Is there any reason for such an exact number for width?

Reply:
> Is there any reason for such an exact number for width?

Copied over the message that's already in use for inline image storybook page. I think the exact number is the actual width of the image, which will be provided by the Chat SDK in real case.